packages/js/src/ui/components/Inbox.tsx

```diff
@@ -1,18 +1,39 @@
 import { createSignal, JSX, Match, Switch } from 'solid-js';
 import { useLocalization } from '../context';
 import { useStyle } from '../helpers';
-import { Bell, Footer, Header, SettingsHeader } from './elements';
+import { Bell, Footer, Header, Settings, SettingsHeader } from './elements';
+import { NotificationList } from './Notification';
 import { Button, Popover } from './primitives';
-import { Settings } from './Settings';
 
 type InboxProps = {
   open?: boolean;
   renderBell?: ({ unreadCount }: { unreadCount: number }) => JSX.Element;
 };
 
-export const Inbox = (props: InboxProps) => {
+const InboxContent = () => {
   const [currentScreen, setCurrentScreen] = createSignal<'inbox' | 'settings'>('inbox');
```

Comment:
Please add these in an const object or an string enum to save some bytes instead of repeating the strings.